File: Program's_Contributed_By_Contributors/Java_Programs/BubbleSort.java
class BubbleSort {
    public static void bubbleSort(int arr[], int n) {                                       
        if (n <= 1) {
            return;
        }

        for (int i=0; i<n-1; i++) {
            if (arr[i] > arr[i+1])  {  
                int temp = arr[i];
                arr[i] = arr[i+1];
                arr[i+1] = temp;
            }
        }

        bubbleSort(arr, n-1);
    }
}

File: C-Sharp/sort_algorithms/Bubble_sort.cs
using System;

class Program
{
    public static void BubbleSort(int[] arr)
    {
        int n = arr.Length;
        for (int i = 0; i < n - 1; i++)
        {
            for (int j = 0; j < n - i - 1; j++)
            {
                if (arr[j] > arr[j + 1])
                {
                    int temp = arr[j];
                    arr[j] = arr[j + 1];
                    arr[j + 1] = temp;
                }
            }
        }
    }

    static void Main()
    {
        int[] arr = { 64, 34, 25, 12, 22, 11, 90 };

        Console.WriteLine("Original Array:");
        Console.WriteLine(string.Join(", ", arr));

        BubbleSort(arr);

        Console.WriteLine("Sorted Array:");
        Console.WriteLine(string.Join(", ", arr));
    }
}
